app/controllers/terms_controller.rb

```diff
@@ -0,0 +1,23 @@
+class TermsController < ApplicationController
```

Comment:
missing? module Publish
```suggestion
module Publish
class TermsController < ApplicationController
```

Reply:
done